sample-code/java/src/IOSBrowserSaucelabsTest.java

```diff
@@ -0,0 +1,48 @@
+import java.net.MalformedURLException;
+import java.net.URL;
+
+import org.openqa.selenium.remote.DesiredCapabilities;
+import org.testng.Assert;
+import org.testng.annotations.AfterTest;
+import org.testng.annotations.BeforeTest;
+import org.testng.annotations.Test;
+
+import io.appium.java_client.ios.IOSDriver;
+import io.appium.java_client.remote.MobileCapabilityType;
+
+/**
+ * IOS Browser Sauce Labs Test.
+ */
+public class IOSBrowserSaucelabsTest extends BaseTest {
+    public static final String USERNAME = "YOUR_USERNAME";
+    public static final String ACCESS_KEY = "YOUR_ACESS_KEY";
+    public static final String URL = "https://"+USERNAME+":" + ACCESS_KEY + "@ondemand.saucelabs.com:443/wd/hub";
+    public static IOSDriver<?> mobiledriver;
+
+    @BeforeTest
+    public void beforeTest( ) throws MalformedURLException {
+        DesiredCapabilities capabilities = new DesiredCapabilities();
+
+        capabilities.setCapability(MobileCapabilityType.APPIUM_VERSION, "1.7.2");
```

Comment:
Same here

Reply:
Removed Appium Version and changed AutomationName to UIAutomator2